tailwind.config.js

```diff
@@ -20,9 +20,11 @@ module.exports = {
             },
             width: {
                 sidebar: "200px",
+                spriteimage: "250px",
```

Comment:
Is there a specific reason why this should be registered in the Tailwind config?